src/Perpetuum/Services/EventServices/EventProcessors/NpcStateAnnouncer.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Perpetuum.Accounting.Characters;
+using Perpetuum.Collections;
+using Perpetuum.Services.Channels;
+using Perpetuum.Services.EventServices.EventMessages;
+using Perpetuum.Zones.NpcSystem.Flocks;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Perpetuum.Services.EventServices.EventProcessors
+{
+    public class NpcStateAnnouncer : EventProcessor
+    {
+        private readonly IChannelManager _channelManager;
+        private const string SENDER_CHARACTER_NICKNAME = "[OPP] Announcer";
+        private const string CHANNEL = "Syndicate Radio";
+        private readonly Character _announcer;
+        private readonly IFlockConfigurationRepository _flockConfigReader;
+        private readonly IDictionary<string, object> _nameDictionary;
+
+        public NpcStateAnnouncer(IChannelManager channelManager, IFlockConfigurationRepository flockConfigurationRepo, ICustomDictionary customDictionary)
+        {
+            _announcer = Character.GetByNick(SENDER_CHARACTER_NICKNAME);
+            _channelManager = channelManager;
+            _flockConfigReader = flockConfigurationRepo;
+            _nameDictionary = customDictionary.GetDictionary(0);
+            _channelManager.JoinChannel(CHANNEL, _announcer);
+            _channelManager.SetMemberRole(CHANNEL, _announcer, ChannelMemberRole.Operator);
+        }
+
+        private string GetNpcName(NpcStateMessage msg)
+        {
+            var config = _flockConfigReader.Get(msg.FlockId);
+            var nameToken = config.EntityDefault.Name + "_name";
+            var name = _nameDictionary[nameToken]?.ToString();
+            return name ?? string.Empty;
+        }
+
+        private IList<string> _aliveMessages = new List<string>()
+        {
+            "has spawned!",
+            "has appeared on Syndicate scanners",
+            "has been detected"
+        };
+
+        private IList<string> _deathMessages = new List<string>()
+        {
+            "has been defeated",
+            "is no longer a threat to Syndicate activity",
+            "'s signature is no longer detected at this time"
```

Comment:
Won't this end up looking a little wonky in the final message due to the whitespace?
(L73 `return $"{npcName} {stateMessage}";`)

Example:
`ValuableNPC 's signature is ...`
-------------^
Vs what it should be
`ValuableNPC's signature is ...`

There's also the case of an npc name ending with an s character which makes this look odd too.

Reply:
I'm leaving this for a `good first issue` enhancement for later.
There is some existing code for using strings stored in the DB to get templated.
A TODO note was added which will later become an issue.